gateway_code/open_nodes/node_fox.py

```diff
@@ -33,9 +33,9 @@ class NodeFox(NodeOpenOCDBase):
     TYPE = 'fox'
     TTY = '/dev/iotlab/ttyON_FOX'
     BAUDRATE = 500000
-    OPENOCD_PATH = 'openocd'  # Use old openocd
-    OPENOCD_CFG_FILE = static_path('iot-lab-fox.cfg')
-    OPENOCD_OPTS = ('target/stm32f1x.cfg',)
+    OPENOCD_PATH = '/opt/openocd-dev/bin/openocd'
+    OPENOCD_CFG_FILE = static_path('iot-lab.cfg')
+    OPENOCD_OPTS = (static_path('iot-lab-fox.cfg'),)
```

Comment:
Was strange that 'opts' was the more generic one before, so flipping it now makes really sense.